pkg/apis/serving/v1alpha2/configmap.go

```diff
@@ -62,11 +66,17 @@ type TransformersConfig struct {
 	Feast TransformerConfig `json:"feast,omitempty"`
 }
 
+// +k8s:openapi-gen=false
+type CommonConfig struct {
```

Comment:
Can we name it `DeploymentConfig`? we have other configuration can be moved here such as autoscaling `TargetConcurrency` and container limits.

Reply:
Sure, happy to change~